cmake/NSIS.template.in

```diff
@@ -29,8 +29,6 @@
   !define MUI_ICON "@CPACK_NSIS_MUI_ICON@"
   !define MUI_UNICON "@CPACK_NSIS_MUI_UNIICON@"
   !define MUI_UI_COMPONENTSPAGE_SMALLDESC
-  !define MUI_LICENSEPAGE_CHECKBOX
-  !define MUI_LICENSEPAGE_CHECKBOX_TEXT "I agree to the license terms."
```

Comment:
Does this change the license page? If yes, how (might be good, just want to understand)? Do we need to update the docs screenshots?

Reply:
it's been commented out for some time. I updated the docs at that time. I am now formally removing the lines.

Reply:
Ah so ! is a comment? :D

Reply:
no there is a commented line earlier that disabled these. `!` is an include directive. `;` and `#` are comments for NSIS.